python/sdk/prelude_sdk/controllers/partner_controller.py

```diff
@@ -139,3 +138,15 @@ def observed_detected(self, test_id: str | None, hours: int | None):
         if res.status_code == 200:
             return res.json()
         raise Exception(res.text)
+
+    @verify_credentials
+    def list_advisories(self, partner: Control):
+        """ Get advisory reports provided by a partner """
+        res = self._session.get(
+            f'{self.account.hq}/partner/advisories/{partner.name}',
```

Comment:
isn't this the value not the name?

Reply:
can be either. name is friendlier so it's usually the one i test

Reply:
oh right I keep forgetting those enums are magic